java/src/test/java/org/rocksdb/InfoLogLevelTest.java

```diff
@@ -113,7 +114,7 @@ public void valueOf() {
    * @throws IOException if file is not found.
    */
   private String getLogContentsWithoutHeader() throws IOException {
-    final String separator = System.getProperty("line.separator");
+    final String separator = Environment.isWindows() ? "\n" : System.getProperty("line.separator");
```

Comment:
No it returns "\r\n". RocksDB port is using native IO functions so that it does not convert "\n" -> "\r\n". Then unit tests fails.